keyboards/mechwild/trisk/keymaps/default/keymap.c

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Kyle McCreery (@kylemccreery)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
+        KC_MUTE, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, RGB_RMOD, RGB_MOD, RGB_TOG
+    )
+};
+
+void keyboard_post_init_user(void) {
+ HSV hsv = {0, 255, 255};
+ RGB rgb = hsv_to_rgb(hsv);
+ for (uint8_t i = 13; i <= 17; i++) {
+            rgb_matrix_set_color(i, rgb.r, rgb.g, rgb.b);
+    }
+};
```

Comment:
```suggestion
}
```
Although, does this just set the underglow LEDs to red (or some other colour)? This could more appropriately be replaced by
```c
bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
  for (uint8_t i = led_min; i <= led_max; i++) {
    if (g_led_config.flags[i] & LED_FLAG_UNDERGLOW) {
      rgb_matrix_set_color(i, RGB_RED);
    }
  }
  return false;
}
```